### ISSUE TITLE
weight decision principle

### ISSUE THREAD

**Issue by Youngsplace0913** [W-zx-Y/DANNet#2]
First of all, thank you for sharing the code of this excellent study.

I would like to give weight by label in different task datasets.
However, the logic of setting the weight of each class was not found in this paper. The supplement paper is not available at present, so there is no way to know.

Can you tell me how you decided by calculating the weight?

Thank you for that.

**Comment by W-zx-Y** [comment]
Hi @Youngsplace0913,

Please check Eq.1 and Eq.2 in Section 3.3 of our paper which is corresponding to the following code.
https://github.com/W-zx-Y/DANNet/blob/89261975c17fe2e72f4b0ebd8ceaab2e138cb862/train.py#L102-L106

**Comment by Youngsplace0913** [comment]
Thank you @W-zx-Y,

I'll check that section, Thank you!
Finally, why do you choose std as 0.05 when training and 0.16 when testing?

**Comment by W-zx-Y** [comment]
Hi @Youngsplace0913,

Sorry for our late reply, this is a busy week for us.
Because we want to find an std value that can achieve the best performance on the validation set. In the paper, we have a figure to show the mIoU varies when different std values are applied on the validation set. After we find the best std (std=0.16 in our experiments) and the corresponding model weight using the validation set, we test the test set and submit the predicted maps to the online server to get the mIoU results. That is why there is an inconsistency between the training and testing. 

So far, we didn't try other std values in the training stage.
